js/scripts.js

```diff
@@ -0,0 +1,64 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const bookList = document.getElementById('booklist');
+
+const addLi = (title, author) => {
+  const li = document.createElement('li');
+  li.innerText = `${title} - ${author}`;
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.innerText = 'Remove';
+  btn.onclick = 'removeBook';
+  li.appendChild(btn);
+  return li;
+};
+
+const refresh = () => {
+  bookList.innerHTML = '';
+  const storedBooks = JSON.parse(localStorage.books);
+  storedBooks.forEach(
+    (book) => {
+      bookList.appendChild(addLi(book.title, book.author));
+    },
+  );
+};
+
+const addBook = () => {
```

Comment:
- [ ]  **Please create a collection that keeps a list of books (hint: you can use an array of objects for that).**

It is one of the mandatory requirements for the project. You have to use a collection to keep a list of all the books. The `addBook` function should additionally add the books to the collection.

Reply:
@smcommits we are already doing it at line 59 `localStorage.setItem('books', JSON.stringify([]));` we are initializing an empty array inside stringify and later we insert objects inside of it. We are just not giving a name to it.